ably-ios/ARTDataQuery+Private.h

```diff
@@ -16,4 +17,10 @@ ART_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ARTRealtimeHistoryQuery ()
+
+@property (strong, readwrite) ARTRealtimeChannel *realtimeChannel;
```

Comment:
The `query` can retain the channel if you create the `ARTRealtimeHistoryQuery` in other part of the main code and the client tries to release the channel.